lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -525,3 +525,10 @@
     pkg:
       - libsqlite3-dev
   when: full_provision
+
+- name: Set core dump pattern
+  ansible.posix.sysctl:
+    name: kernel.core_pattern
```

Comment:
This will affect not only containerized applications, right?

Reply:
yes - but I choose a directory that is writeable from inside and outside of the container so it should be fine

Reply:
Is this for non-container setups setup in line 85 "Copy sysctl file for core dumps"? There the pattern seems to be defined slightly different (additionally to the path).

Reply:
Right, we should configure that in only one place. Currently it is `|/usr/local/bin/coredump core-%t-%e-%p` -  I suppose with containers we cannot pipe it like that but need to define a folder. Personally I would prefer a directory without `docker`, as I find it a bit confusing if you have to look for coredumps in `/something/docker` even when not using docker at all.

Reply:
(fyi, if relevant for a solution here: iiuc then this role is not running in production setups - there `99-magma.conf` is deployed via `magma.deb`)

Reply:
I think  I will replace the coredump script inside of the container so both solutions can be maintained

Reply:
Ok turns out you do not need any special logic at all to capture cores on the dev vm:
```
vagrant@magma-dev:~/magma/lte/gateway/docker$ docker-compose exec sessiond /bin/bash
root@magma-dev:/# ./crash 
Segmentation fault (core dumped)
root@magma-dev:/# exit
exit
vagrant@magma-dev:~/magma/lte/gateway/docker$ ls /var/core
core-1661521523-a.out-533.tgz  core-1661521584-crash-558.tgz
vagrant@magma-dev:~/magma/lte/gateway/docker$ tar tzf /var/core/core-1661521584-crash-558.tgz
./
./core-1661521584-crash-558.gz
./syslog
./mme.log
./version
```

Reply:
You just have to know where to look.